.github/workflows/build-release.yaml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build-release:
-    name: Build and release
+    name: Release to npm
```

Comment:
I liked more the previous one `Build and Release`. We are not really releasing _**to**_ npm